js/app/components/timer.tsx

```diff
@@ -0,0 +1,40 @@
+import { View, Text } from "tamagui";
+import { useState, useEffect } from "react";
+
+export default function Timer({ start }: { start: string | Date }) {
+  const [elapsedTime, setElapsedTime] = useState(0);
+
+  useEffect(() => {
+    const startDate = typeof start === "string" ? new Date(start) : start;
+
+    const interval = setInterval(() => {
+      const now = new Date();
+      const elapsed = Math.floor((now.getTime() - startDate.getTime()) / 1000);
+      setElapsedTime(elapsed);
+    }, 1000);
```

Comment:
If it's at 1 second it'll occasionally hit a case where it skips a second because the `setInterval` went off on either side of it. Let's do:

```suggestion
    }, 250);
```